resources/install.rb

```diff
@@ -179,6 +179,11 @@
         baseurl      repo_url
         gpgkey       repo_signing_key
       end
+      execute 'disable-nginx-module' do
+        command 'dnf -y module disable nginx'
```

Comment:
I think this is a better way to do this.
```suggestion
      execute 'dnf -y module disable nginx' do
```